myhdl/test/core/test_ShadowSignal.py

```diff
@@ -165,8 +166,9 @@ def check():
         yield delay(10)
         assert s == c
         c.next = None
+        d.next = 0
         yield delay(10)
-        assert s == None
+        assert s == d
```

Comment:
Why were these changes made?  You should back out these changes since the PR is specific to expanding the enum test coverage.